sklearn/metrics/ranking.py

```diff
@@ -682,6 +685,7 @@ def label_ranking_average_precision_score(y_true, y_score):
     check_consistent_length(y_true, y_score)
```

Comment:
This is duplicate. Please remove this line and move ``check_consistent_length(y_true, y_score, sample_weight)`` here.